src/App/App.js

```diff
@@ -1,24 +1,40 @@
 import React, { Component } from 'react';
 import MoviesContainer from '../MoviesContainer/MoviesContainer';
 import Header from '../Header/Header';
+import SingleMovie from '../SingleMovie/SingleMovie';
 import movieData from '../movie-data';
 import './App.css';
 
 class App extends Component {
   constructor () {
     super();
     this.state = {
-      movies: movieData.movies
+      movies: movieData.movies,
+      movieDetails: [],
     }
   }
+  
+  displayMovieDetails = (id) => {
+    const singleMovie = this.state.movies.filter(movie => movie.id === id);
```

Comment:
Grateful for help to realize that find wasn't working because it returns and element, not an array!